.github/workflows/main.yaml

```diff
@@ -183,5 +184,8 @@ jobs:
       - name: Install dependencies
         run: poetry install
 
+      - name: Install dependencies with conda for circleci
```

Comment:
is this step necessary?
circleci cli should be already installed via conda/dev.yaml

Reply:
Right, I'm going to remove it, thank you!